src/window.ui

```diff
@@ -2,58 +2,45 @@
 <interface>
   <requires lib="gtk+" version="3.24" />
   <requires lib="libhandy" version="1.0" />
-  <object class="HdyApplicationWindow" id="window">
+  <object class="AdwApplicationWindow" id="window">
     <property name="resizable">True</property>
     <property name="default_width">560</property>
     <property name="default_height">240</property>
-    <property name="type_hint">dialog</property>
-    <property name="urgency_hint">True</property>
+    <property name="title">Commit</property>
     <child>
       <object class="GtkBox">
-        <property name="visible">True</property>
         <property name="orientation">vertical</property>
         <child>
-          <object class="HdyHeaderBar" id="header">
-            <property name="visible">True</property>
-            <property name="title">Commit</property>
+          <object class="AdwHeaderBar" id="header">
+            <property name="show_start_title_buttons">False</property>
+            <property name="show_end_title_buttons">False</property>
             <child>
               <object class="GtkButton" id="cancelButton">
                 <property name="label" translatable="yes">Cancel</property>
-                <property name="visible">True</property>
-                <property name="receives_default">True</property>
```

Comment:
Looks like it's broken on `main` but the intention is for the cancel button to have default focus.

Reply:
https://github.com/sonnyp/Commit/issues/33